packages/common.vm/tools/vm.common/vm.common.psm1

```diff
@@ -1288,6 +1288,16 @@ public class Shell {
     }
 }
 
+# Sort Desktop icons by item type using WScript.Shell to replicate the manual steps
+function VM-Sort-Desktop-Icons {
+    VM-Write-Log "INFO" "Sorting Desktop icons"
+    (New-Object -ComObject Shell.Application).toggleDesktop();
+    Start-Sleep -Milliseconds 100;
+    $objShell = New-Object -ComObject WScript.Shell;
+    $objShell.SendKeys("^{ }+{F10}oi");
```

Comment:
This didn't seem to always work depending on what might have been in focus before executing. I tried it with a folder having been selected before switching to the powershell terminal and executing the command and also with a folder not selected and ran into varying issues.

I was able to modify the key strokes to this and it seems to work more consistently:

Reply:
doesn't Control+space (`^{ }`) unselect the folder for you?

Reply:
I see, if there are several folders selected it does run into issues

Reply:
I'll update the code, good catch @emtuls!